[2021-12-29 19:58:25,743] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-12-23T00:00:00+00:00 [queued]>
[2021-12-29 19:58:25,748] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-12-23T00:00:00+00:00 [queued]>
[2021-12-29 19:58:25,748] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 19:58:25,748] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-29 19:58:25,748] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-29 19:58:25,771] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): runme_2> on 2021-12-23T00:00:00+00:00
[2021-12-29 19:58:25,773] {standard_task_runner.py:52} INFO - Started process 4356 to run task
[2021-12-29 19:58:25,791] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-12-23T00:00:00+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/sandbox/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmps55aqeq7', '--error-file', '/tmp/tmptn823wpi']
[2021-12-29 19:58:25,792] {standard_task_runner.py:77} INFO - Job 17: Subtask runme_2
[2021-12-29 19:58:25,923] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-12-23T00:00:00+00:00 [running]> on host airflowvm
[2021-12-29 19:58:26,141] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-12-23T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-23T00:00:00+00:00
[2021-12-29 19:58:26,142] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-29 19:58:26,142] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo "example_bash_operator__runme_2__20211223" && sleep 1']
[2021-12-29 19:58:26,151] {subprocess.py:75} INFO - Output:
[2021-12-29 19:58:26,152] {subprocess.py:79} INFO - example_bash_operator__runme_2__20211223
[2021-12-29 19:58:27,156] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-29 19:58:27,200] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20211223T000000, start_date=20211229T195825, end_date=20211229T195827
[2021-12-29 19:58:27,254] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-29 19:58:27,260] {local_task_job.py:151} INFO - Task exited with return code 0
